index.js

```diff
@@ -212,11 +223,24 @@ module.exports = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	let killedByPid = false;
+
+	spawned._kill = spawned.kill;
+
+	const killSpawned = signal => {
+		if (process.platform === 'win32') {
+			spawned._kill(signal);
+		} else {
+			// Kills the spawned process and its descendents using the pid range hack
+			// Source: https://azimi.me/2014/12/31/kill-child_process-node-js.html
```

Comment:
From reading the article, it looks like the PID trick only works when using `{detached: true}` which we cannot assume.